src/tests/PlayerButton.test.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+
+import PlayerButton from 'components/AudioPlayer/PlayerButtons/PlayerButton';
+
+import icon from 'components/AudioPlayer/icons/fast-forward.svg';
+
+test('it should have correct className, be disabled and should NOT be clickable', () => {
```

Comment:
After reading this spec, I believe the className shouldn't be a passed-thru prop but an internal component state. The button already knows when it is disabled 🤔

Reply:
Not exactly an `useState` but a conditional that happens on render-time.